test.sh

```diff
@@ -24,12 +30,92 @@ function prepareOutputDir() {
   mkdir -p output
 }
 
-function createTestEnvironment() {
+# Build DAP test container & start the cluster
+function createDAPTestEnvironment() {
+  # docker-compose build --pull client-dap cuke-master postgres test-dap
+  docker-compose up -d client-dap cuke-master postgres test-dap
+
+  # Delay to allow time for conjur to come up
+  echo 'Waiting for conjur server to be healthy'
+  docker-compose run --rm test ./wait_for_server.sh
+}
+
+function loadDAPTestPolicy() {
+  echo '-----------------------test.sh------------------------------'
+  echo "Loading DAP test policy"
+  echo '------------------------------------------------------------'
+
+  # get DAP container id
+  dap_client_cid=$(docker-compose ps -q client-dap)
+
+  ssl_cert=$(docker-compose exec cuke-master cat /opt/conjur/etc/ssl/conjur.pem)
+  export CONJUR_SSL_CERTIFICATE=$(docker-compose exec cuke-master cat /opt/conjur/etc/ssl/conjur.pem)
+
+  docker-compose up -d client-dap
+  docker-compose exec client-dap conjur authn login -u admin -p secret
+
+  # copy test-policy into a /tmp/test-policy within the client container
+  docker cp test-policy ${dap_client_cid}:/tmp
+
+  docker-compose exec client-dap bash -c "conjur policy load root /tmp/test-policy/root.yml"
+}
+
+function initializeDAPCert () {
```

Comment:
there shouldn't be a space between the method and the parentheses for consistency

Reply:
also, it would be great if this method had an announcement like the others do - it does make it easier to see when it's being called in the logs

Reply:
💯